stdlib/utils/src/main/java/org/ballerinalang/utils/Clone.java

```diff
@@ -71,4 +75,16 @@ public void execute(Context ctx) {
         }
         ctx.setReturnValues(refRegVal.copy(new HashMap<>()));
     }
+
+    public static Object clone(Strand strand, Object value) {
+        RefValue refValue = (RefValue) value;
+        if (refValue == null) {
```

Comment:
when can this be null?

Reply:
Value can be null when cloning nil value.